### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by ivinayakg**

### CODE REVIEW

File: src/pages/Login.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  };
+
+  // this function checks wether the click is made on the component or the background
+  const PointerOnContainer = (client, event) => {
+    if (client.left > event.clientX || client.right < event.clientX) {
+      return false;
+    } else if (client.top > event.clientY || client.bottom < event.clientY) {
+      return false;
+    } else return true;
+  };
```
Comment (rahulyadav139) [inline src/pages/Login.js:200]: Great work here.

File: src/pages/Login.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                onClick={() => {
+                  LoginHandler({
+                    email: "hell@gmail.com",
+                    password: "12345678",
+                  }).then((res) => {
+                    setTimeout(() => {
+                      dispatch({ for: "USER", type: "UPDATE" });
+                      closeModal();
+                      navigate(0);
+                    }, 2000);
```
Comment (rahulyadav139) [inline src/pages/Login.js:101]: somewhere you are using async method, somewhere you are using then method. Make a uniformity in the codes.

**Review by rahulyadav139 — APPROVED** [review]

**Review by Maianki — APPROVED** [review]
Small PR nothing much to suggest. You can merge. You have commented your code where it is needed so great work there.

File: src/pages/Login.js (hunk 2 above)
Comment (ivinayakg) [reply src/pages/Login.js:101]: yep you are right will do that

**Pull request merged**